src/main/java/salesforce/utilities/Urls.java

```diff
@@ -1,4 +1,4 @@
-package salesforce;
+package salesforce.utilities;
 
 public enum Urls {
     PATH_LOGIN("https://login.salesforce.com/"),
```

Comment:
Maybe this URLs could be set in your config or env files

Reply:
And don't forget the JavaDocs, review your check style file

Reply:
I user a lot of url, it is not sensitive information